index.md

```diff
@@ -155,7 +155,8 @@ filtered_df.summary().show()
             <div class="tab-pane fade" id="nav-sql" role="tabpanel" aria-labelledby="nav-sql-tab">
                 <div class="mb-2 title">Run now</div>
                 <div class="code">
-                    <p>$ SPARK-HOME/bin/spark-sql</p>
+                    <p>$ docker pull apache/spark</p>
+                    <p>$ docker run -it apache/spark /opt/spark/bin/spark-sql</p>
```

Comment:
For simplicity, we can use `docker run -ti` directly, because it will automatically download and complete the `docker pull` process

Reply:
nice, will update